chromedriver_autoinstaller/utils.py

```diff
@@ -275,7 +285,10 @@ def download_chromedriver(path: Optional[AnyStr] = None, no_ssl: bool = False):
     if path:
         if not os.path.isdir(path):
             raise ValueError(f"Invalid path: {path}")
-        chromedriver_dir = os.path.join(os.path.abspath(path), major_version)
+        if make_version_dir:
+            chromedriver_dir = os.path.join(os.path.abspath(path), major_version)
+        else:
+            chromedriver_dir = os.path.abspath(path)
```

Comment:
This make_version_dir option only applies if path is specified.

Can we include this new option when path is not specified too in the else below?

`    else:
        if make_version_dir:
            chromedriver_dir = os.path.join(
                os.path.abspath(os.path.dirname(__file__)), major_version
            )
        else:
            chromedriver_dir = os.path.abspath(os.path.dirname(__file__))`